HousingResidentInformationAPI/build.sh

```diff
@@ -1,21 +1,21 @@
-#!/bin/bash
-
-#install zip on debian OS, since microsoft/dotnet container doesn't have zip by default
-if [ -f /etc/debian_version ]
-then
-  apt -qq update
-  apt -qq -y install zip
-fi
-
-#dotnet restore
-dotnet tool install --global Amazon.Lambda.Tools --version 4.0.0
-
-
-# (for CI) ensure that the newly-installed tools are on PATH
-if [ -f /etc/debian_version ]
-then
-  export PATH="$PATH:/$(whoami)/.dotnet/tools"
-fi
-
-dotnet restore
-dotnet lambda package --configuration release --framework netcoreapp3.1 --output-package ./bin/release/netcoreapp3.1/uh-resident-information-api.zip
+#!/bin/bash
+
+#install zip on debian OS, since microsoft/dotnet container doesn't have zip by default
+if [ -f /etc/debian_version ]
+then
+  apt -qq update
+  apt -qq -y install zip
+fi
+
+#dotnet restore
+dotnet tool install --global Amazon.Lambda.Tools --version 4.0.0
+
+
+# (for CI) ensure that the newly-installed tools are on PATH
+if [ -f /etc/debian_version ]
+then
+  export PATH="$PATH:/$(whoami)/.dotnet/tools"
+fi
+
+dotnet restore
+dotnet lambda package --configuration release --framework netcoreapp3.1 --output-package ./bin/release/netcoreapp3.1/Housing-resident-information-api.zip

```

Comment:
housing-resident-information-api needs to be with lower case "h", as this is what the serverless.yml file is looking for